name.abuchen.portfolio/src/name/abuchen/portfolio/util/TradeCalendarManager.java

```diff
@@ -345,6 +366,37 @@ public class TradeCalendarManager
         tc.add(fixed(SECOND_CHRISTMAS_DAY, Month.DECEMBER, 26));
         CACHE.put(tc.getCode(), tc);
 
+        
+        // Tel Aviv Stock Exchange starting 2026
+        // https://www.tase.co.il/en/content/knowledge_center/trading_vacation_schedule#vacations
+        tc = new TradeCalendar("tlv", Messages.LabelTradeCalendarTLV, STANDARD_WEEKEND); //$NON-NLS-1$
```

Comment:
What not use the TLV_WEEKEND here?
It is defined above, but then not used here.
I see the test differs between weekends "before 2016" and "after 2016"